.golangci.yml

```diff
@@ -119,3 +119,9 @@ issues:
     - linters: [revive]
       text: "^if-return: "
       path: error_writer.go
+    # We want to set http.Server's logger
+    - linters: [forbidigo]
+      path: internal/memhttp
```

Comment:
Can we restrict this bypass to just logger-related failures? I'd still like to forbid panic, FIXME, etc. in `memhttp`. I think the right config key is `text:` (we use it in the revive bypass a little further above).